server/services/task_annotations_service.py

```diff
@@ -13,7 +13,8 @@ def add_or_update_annotation(annotation, project_id, annotation_type):
 
         task_id = annotation['taskId']
         source = annotation['annotationSource'] if 'annotationSource' in annotation.keys() else None
-        task_annotation = TaskAnnotation(task_id, project_id, annotation_type, source, annotation['properties'])
+        markdown = annotation['annotationMarkdown'] if 'annotationMarkdown' in annotation.keys() else None
```

Comment:
Can this be changed to:

markdown = annotation.get('annotationMarkdown', None)